From a2ae076a2c2d867f6ced3b4b97b507105903c196 Mon Sep 17 00:00:00 2001
From: Kirill Kapranov <kirill.kapranov@compulab.co.il>
Date: Wed, 11 Sep 2019 19:55:41 +0300
Subject: [PATCH 34/36] Move usb-eth dongle asix-88179 driver into kernel

The driver has been compiled into the kernel (instead a module) in order to
provide NFS rootfs over the dongle.

Signed-off-by: Kirill Kapranov <kirill.kapranov@compulab.co.il>
---
 arch/arm64/configs/ucm-imx8m-mini_defconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/configs/ucm-imx8m-mini_defconfig b/arch/arm64/configs/ucm-imx8m-mini_defconfig
index 5d7108a79189..bb8bc3d23baa 100644
--- a/arch/arm64/configs/ucm-imx8m-mini_defconfig
+++ b/arch/arm64/configs/ucm-imx8m-mini_defconfig
@@ -200,7 +200,8 @@ CONFIG_REALTEK_PHY=m
 CONFIG_USB_PEGASUS=m
 CONFIG_USB_RTL8150=m
 CONFIG_USB_RTL8152=m
-CONFIG_USB_USBNET=m
+CONFIG_USB_USBNET=y
+CONFIG_USB_NET_AX88179_178A=y
 CONFIG_USB_NET_DM9601=m
 CONFIG_USB_NET_SR9800=m
 CONFIG_USB_NET_SMSC75XX=m
-- 
2.11.0

